djangocms_icon/static/djangocms_icon/js/libs/bootstrap/popover.js

```diff
@@ -0,0 +1,108 @@
+/* ========================================================================
```

Comment:
no way to load this via npm instead of adding the file here?

Reply:
i was actually thinking i can't import separate files from bootstrap-sass package but runkit suggests otherwise, will do